data-syncs/building-data-syncs/README.md

```diff
@@ -1,161 +1,138 @@
-# Build data syncs
+# Build data syncs in Cinchy
 
-## Guide
+Data syncs in Cinchy synchronizes information between different systems. This guide walks you through the process of setting up your data syncs.
 
-You can use this checklist in conjunction with the documentation below and elsewhere in this space to configure your data syncs in Cinchy.
+## Basic workflow
 
-- [ ] [Install any applicable components](../installation-and-maintenance/)
-- [ ] Determine your [Data Sync Type](types-of-data-syncs.md) and [Design Pattern](common-design-patterns.md)
-- [ ] Define and Configure your [Data Sync Source](../supported-data-sync-sources/)
-- [ ] Define and Configure your [Data Sync Destination](../supported-data-sync-destinations/)
-- [ ] Define and Configure your [Sync Actions](sync-actions.md)
-- [ ] Define and Configure your [Event Stream source](../supported-real-time-sync-stream-sources/) (if running a real-time sync)
+The basic workflow of a data sync in Cinchy is:
 
-## Prerequisites
+1. [Install Necessary Components](link-to-installation-doc).
```

Comment:
Are these linkes properly replaced later?

Reply:
This is a draft. This section should be updated once all the moving around is complete. I've added a markdown-link-check on every push to make sure it doesn't pass me be.